components/decentralized_dns/constants.h

```diff
@@ -8,12 +8,40 @@
 
 namespace decentralized_dns {
 
+enum class Provider {
+  UNSTOPPABLE_DOMAINS,
+  ENS,
+};
+
 enum class ResolveMethodTypes {
   ASK,
   DISABLED,
   DNS_OVER_HTTPS,
+  ETHEREUM,
+};
+
+enum class RecordKeys {
+  DWEB_IPFS_HASH,
+  IPFS_HTML_VALUE,
+  DNS_A,
+  DNS_AAAA,
+  BROWSER_REDIRECT_URL,
+  IPFS_REDIRECT_DOMAIN_VALUE,
+  MAX_RECORD_KEY = IPFS_REDIRECT_DOMAIN_VALUE,
 };
 
+// Need to match RecordKeys above.
+const constexpr char* const kRecordKeys[] = {
```

Comment:
I would add a comment about what is this about

Reply:
1cdb4f081decc67ac46ac53ed9bee83fcf3ebb34